### ISSUE TITLE
Android atlas sample with wrong textures

### ISSUE THREAD

**Issue by devemux86** [mapsforge/vtm#311]
Trying `AtlasMarkerOverlayActivity` and modifying the symbols position like:
```java
symbol = new MarkerSymbol(regionsMap.get("poi"), HotspotPlace.UPPER_RIGHT_CORNER);
mFocusMarker = new MarkerSymbol(regionsMap.get("focus"), HotspotPlace.LOWER_LEFT_CORNER);
```
- At start we see double regular symbols (screenshot 1)
- On tap we see simultaneously the regular and the focus symbols (screenshot 2)

![1](https://cloud.githubusercontent.com/assets/3484020/23462523/d736e5a2-fe96-11e6-824c-e8d034e3407c.png) ![2](https://cloud.githubusercontent.com/assets/3484020/23462524/d7648502-fe96-11e6-98a4-4747379b9097.png)

**Comment by devemux86** [comment]
On desktop things seem working fine.

@Longri can you have a look?

**Comment by Longri** [comment]
Shure, but not today

**Comment by devemux86** [comment]
It's worth mentioning that:
- The `MarkerOverlayActivity` which doesn't use texture atlas works correctly
- The fix in #303 does not solve this issue

**Comment by devemux86** [comment]
Oh the error is irrelevant to texture atlas.
More because of to wrong activity inheritance.
